libkbfs/config_local.go

```diff
@@ -643,7 +653,7 @@ func (c *ConfigLocal) resetCachesWithoutShutdown() DirtyBlockCache {
 	c.kcache = NewKeyCacheStandard(defaultMDCacheCapacity)
 	c.kbcache = NewKeyBundleCacheStandard(defaultMDCacheCapacity * 2)
 	// Limit the block cache to 10K entries or 1024 blocks (currently 512MiB)
-	c.bcache = NewBlockCacheStandard(10000, MaxBlockSizeBytesDefault*1024)
+	c.bcache = NewBlockCacheStandard(10000, getDefaultCleanBlockCacheCapacity())
```

Comment:
Can you please print out what you're setting it to?  You can use `c.MakeLogger().CDebugf(nil, ...)`.